decidim-initiatives/spec/shared/create_initiative_example.rb

```diff
@@ -144,53 +110,6 @@
           expect(initiative.signature_end_date).to be_nil
         end
       end
-
-      context "when the initiative type enables custom signature end date" do
-        let(:initiative_type) { create(:initiatives_type, :custom_signature_end_date_enabled) }
-
-        let(:form_params) do
-          {
-            title: "A reasonable initiative title",
-            description: "A reasonable initiative description",
-            type_id: scoped_type.type.id,
-            signature_type: "online",
-            scope_id: scoped_type.scope.id,
-            decidim_user_group_id: nil,
-            signature_end_date: Date.tomorrow
-          }
-        end
-
-        it "sets the signature end date" do
-          command.call
-          initiative = Decidim::Initiative.last
-
-          expect(initiative.signature_end_date).to eq(Date.tomorrow)
-        end
-      end
-
-      context "when the initiative type enables area" do
```

Comment:
Any reason why we're removing this example?

Reply:
This is being removed because the those particular steps are just for CreateInitiative command which is being used only for the first step in the wizard. After that, the UpdateInitiative command is being used.